stylesheets/_font_stack.scss

```diff
@@ -35,10 +27,7 @@ $NTA-Light-Tabular:
        local("Helvetica");
 }
 
-$Helvetica-Regular:
-  "GDS-Logo",
-  sans-serif;
+$Helvetica-Regular: "GDS-Logo", sans-serif;
```

Comment:
It might help to move this above, to sit with the other Sass variables for font stacks.